spec/features/audit_trail_spec.rb

```diff
@@ -1,14 +1,14 @@
 require 'rails_helper'
 
-feature 'Audit trail' do
+describe 'Audit trail' do
   include PermittedDomainHelper
 
   let(:person) { create(:super_admin, email: 'test.user@digital.justice.gov.uk') }
   before do
     token_log_in_as person.email
   end
 
-  scenario 'Auditing an edit of a person' do
+  it 'Auditing an edit of a person' do
```

Comment:
Just interested! What's the background to this change and the one above? feature/describe and scenario/it...

Reply:
I believe these were things that Rubocop pulled up - I'm not sure the exact reasoning. I think it should be reworded though as it doesn't make much sense in its current form as an `it`.